chart/githubconnector/README.md

```diff
@@ -1,15 +1,35 @@
-# github-connector
+# Github Connector <!-- omit in toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Installation in Kyma with Helm](#installing-in-kyma-using-helm)
+	- [Prerequisites](#prerequisites-1)
+	- [Steps](#steps)
 
 ## Overview
-The chart creates a github connector deployment, and a namespace inside kyma.
-Moreover it creates a service, binds it to the newly created namespace and exposes its API. Apart from that it creates an application that for now, the user has to manually bind to the namespace
+The chart creates a GitHub Connector Deployment, and a Namespace inside Kyma.
```

Comment:
```suggestion
The chart creates a GitHub Connector Deployment and a Namespace inside Kyma.
```
No need to put a comma before "and" unless there are more than 2 objects, e.g. "I like bananas, mango, kiwi, and whisky".